khelo/lib/ui/flow/home/home_screen.dart

```diff
@@ -187,20 +211,29 @@ class _HomeScreenState extends ConsumerState<HomeScreen>
     BuildContext context,
     MatchModel match,
   ) {
-    return Column(
+    return Row(
```

Comment:
No because we also show dot container.

Reply:
that can be added in WidgetSpan right?

Reply:
No Because if i used Text.rich then dot not how center.

![dot](https://github.com/canopas/khelo/assets/140152611/55514b6a-5504-4180-b76e-8861393009f1)